### PR TITLE
OCPBUGS-21751: update ip_whitelist entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OCPBUGS-21751

Link to docs preview: https://66299--docspreview.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration#nw-route-specific-annotations_route-configuration

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@melvinjoseph86 could you PTAL for QE ack? thank you!

File: modules/nw-route-specific-annotations.adoc [hunk 1]
```diff
@@ -39,9 +39,10 @@ Note: Using this annotation provides basic protection against distributed denial
 |`ingresses.config/cluster ingress.operator.openshift.io/hard-stop-after` | You can set either an IngressController or the ingress config . This annotation redeploys the router and configures the HA proxy to emit the haproxy `hard-stop-after` global option, which defines the maximum time allowed to perform a clean soft-stop. | `ROUTER_HARD_STOP_AFTER`
 |`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`haproxy.router.openshift.io/ip_whitelist`
-| Sets a whitelist for the route. The whitelist is a space-separated list of IP addresses and CIDR ranges for the approved source addresses. Requests from IP addresses that are not in the whitelist are dropped.
+| Sets an allowlist for the route. The allowlist is a space-separated list of IP addresses and CIDR ranges for the approved source addresses. Requests from IP addresses that are not in the allowlist are dropped.
+
+The maximum number of IP addresses and CIDR ranges in an allowlist that can be written to `haproxy.config` is 64. [^1^] |
```
Comment (melvinjoseph86) [inline modules/nw-route-specific-annotations.adoc:44]: i tested in 4.15 build the same and the whitelist which will be shown is 61 only and not 64. i would prefer the wordings of the sentence as `The maximum number of IP addresses and CIDR ranges directly visible in the`haproxy.config` is 61.`

File: modules/nw-route-specific-annotations.adoc [hunk 2]
```diff
@@ -66,6 +67,15 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 `if-none`: sets the header if it is not already set.| `ROUTER_SET_FORWARDED_HEADERS`
 
 |===
+[.small]
+--
+1. If the number of IP addresses and CIDR ranges in an allowlist exceeds 64, they are written into a separate file that is then referenced from `haproxy.config`.
```
Comment (melvinjoseph86) [inline modules/nw-route-specific-annotations.adoc:72]: If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from `haproxy.config`. The file will be stored in `/var/lib/haproxy/router/whitelists` folder

File: modules/nw-route-specific-annotations.adoc [hunk 3]
```diff
@@ -66,6 +67,15 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
...
++
+[NOTE]
+====
+To ensure that the addresses are written to the allowlist, check that the full list of CIDR ranges are listed in the Ingress Controller configuration file. The etcd object size limit restricts how large a route annotation can be. Because of this, it creates a threshold for the maximum number of IP addresses and CIDR ranges that you can include in an allowlist.
+====
+--
```
Comment (melvinjoseph86) [inline modules/nw-route-specific-annotations.adoc:78]: my suggestion is to add this portion just after line 44 to have a continuation.

Comment (jldohmann) [reply modules/nw-route-specific-annotations.adoc:78]: The technical limitations of how tables are currently rendering makes content unreadable inside a cell, but I can take a look at a later time whenever the [issue](https://github.com/redhataccess/ascii_binder/issues/184) is resolved

**Pull request merged**